source/algorithms.tex

```diff
@@ -8724,19 +8724,20 @@
 \end{itemdecl}
 
 \begin{itemdescr}
+\pnum
+Let \tcode{comp} be \tcode{less\{\}}
```

Comment:
I'm not sure whether this part is editorial for legacy `std` versions. Since `less{}` is more determinate than the built-in operation when the comparison between pointers is involved.

Reply:
Yeah this part seems like a normative change, requiring a total order for pointers where previously it was unspecified what you got for unrelated pointers.

Reply:
Yes, `less` is more.

Reply:
@jwakely Should I file this change as an LWG issue, or should we simply have LWG sign off on the PR?

Reply:
Thanks, both! @CaseyCarter, could you close this PR then once the LWG issue exists?

Reply:
I've submitted https://github.com/cplusplus/LWG/pull/434 with these changes.

Reply:
> I've submitted [cplusplus/LWG#434](https://github.com/cplusplus/LWG/pull/434) with these changes.

The PR has merged.